### PR TITLE
Add an example of mDNS candidate

### PR DESCRIPTION
**Pull request by tQsW**
PR for #48

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -226,12 +226,23 @@ not have any effect on connectivity or connection setup time.
...
+The first example illustrates an mDNS candidate gathered with a version 4 UUID
+name. The second example shows the typical case of exchanging mDNS candidates,
+and other two examples demonstrate how peer-reflexive candidates for local IP
+addresses can be created due to timing differences.
 
-Normal handling
+mDNS Candidate
+--------------
+
+The exchange of ICE candidates relies on out-of-band signaling, and we
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: Might want to combine this into the example below, basically show the candidates exchanged by ICE Agent 1 and ICE Agent 2 in a list below the drawing.

Comment (tQsW) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: Came up with two options for the presentation of the example. Let me know which one reads better.

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 2]
```diff
@@ -226,12 +226,12 @@ not have any effect on connectivity or connection setup time.
 Examples
 ========
 
-The examples below show how the mDNS technique is used during ICE
-processing. The first example shows the typical case, and the 
-other two examples demonstrate how peer-reflexive candidates for
-local IP addresses can be created due to timing differences.
+The examples below show how the mDNS technique is used during ICE processing.
+The first example shows the typical case, and other two examples demonstrate
```
Comment (youennf) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: Maybe just me, but it reads better with 'the' than without.

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 3]
```diff
@@ -252,6 +252,17 @@ normally to obtain the corresponding IP addresses.
                           |===== STUN check to 2.2.2.2 ====>|
                           |                                 |
 
+The exchange of ICE candidates relies on out-of-band signaling, for instance
+the SDP Offer/Answer procedure defined in {{ICESDP}}. In the above example and
+also the following ones, the candidate attributes in the SDP messages to
```
Comment (youennf) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: s/and also the following ones//

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 4]
```diff
@@ -252,6 +252,17 @@ normally to obtain the corresponding IP addresses.
...
+exchange the mDNS candidates between ICE Agent 1 and 2 can be given by:
+
+* For ICE Agent 1,
+a=candidate:1 1 udp 2122262783 1f4712db-ea17-4bcf-a596-105139dfd8bf.local 54596 typ host
+
+* For ICE Agent 2,
+a=candidate:1 1 udp 2122262783 2579ef4b-50ae-4bfe-95af-70b3376ecb9c.local 61606 typ host
+
```
Comment (youennf) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]: I think I prefer this option

File: draft-ietf-rtcweb-mdns-ice-candidates.md (hunk 2 above)
Comment (juberti) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: agreed, seems like this was accidental

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 5]
```diff
@@ -252,6 +252,17 @@ normally to obtain the corresponding IP addresses.
                           |===== STUN check to 2.2.2.2 ====>|
                           |                                 |
 
+The exchange of ICE candidates relies on out-of-band signaling, for instance
+the SDP Offer/Answer procedure defined in {{ICESDP}}. In the above example and
+also the following ones, the candidate attributes in the SDP messages to
+exchange the mDNS candidates between ICE Agent 1 and 2 can be given by:
+
+* For ICE Agent 1,
+a=candidate:1 1 udp 2122262783 1f4712db-ea17-4bcf-a596-105139dfd8bf.local 54596 typ host
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]:
```suggestion

    a=candidate:1 1 udp 2122262783 1f4712db-ea17-4bcf-a596-105139dfd8bf.local 54596 typ host
```

Comment (juberti) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: (use monospace)

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 6]
```diff
@@ -252,6 +252,17 @@ normally to obtain the corresponding IP addresses.
...
+
+* For ICE Agent 2,
+a=candidate:1 1 udp 2122262783 2579ef4b-50ae-4bfe-95af-70b3376ecb9c.local 61606 typ host
```
Comment (juberti) [inline draft-ietf-rtcweb-mdns-ice-candidates.md]:
```suggestion

    a=candidate:1 1 udp 2122262783 2579ef4b-50ae-4bfe-95af-70b3376ecb9c.local 61606 typ host
```

File: draft-ietf-rtcweb-mdns-ice-candidates.md (hunk 2 above)
Comment (tQsW) [reply draft-ietf-rtcweb-mdns-ice-candidates.md]: Yeah, I deleted the word by accident.
```suggestion
The first example shows the typical case, and the other two examples demonstrate
```

**Comment by tQsW** [comment]
The rendering in "View" does not show the monospace properly. I will figure out if it is a source problem or just the view issue.

**Comment by juberti** [comment]
I think it's a problem with bullet lists and monospace. Probably works if you remove the bullets.

However, I just realized the lines will be too long. You'll need to wrap them, e.g.,

      candidate:1 1 udp 2122262783 1f4712db-ea17-4bc-a596-105139dfd8bf.local
                54596 typ host

**Comment by tQsW** [comment]
Ah, here we go. You are right, and just the monospace lines worked.

**Comment by tQsW** [comment]
Will wait for another approval before merging.

**Comment by youennf** [comment]
LGTM, merging.

**Pull request merged**